cmd/loadTests.go

```diff
@@ -53,6 +55,102 @@ var (
 	threadsWG                          sync.WaitGroup
 )
 
+type LogData struct {
+	Timestamp                         string      `json:"timestamp"`
+	MachineName                       string      `json:"machineName"`
+	BinaryDetails                     string      `json:"binaryDetails"`
+	NumberOfThreads                   int         `json:"Number of threads"`
+	NumberOfUsersPerThread            int         `json:"Number of users per thread"`
+	BatchSize                         int         `json:"Batch size per thread"`
+	NumberOfUsers                     int         `json:"Total number of users"`
+	LoadTestCompletionStatus          string      `json:"loadTestCompletionStatus"`
+	AverageTimeToSpinUpUsers          FloatFormat `json:"Average Time taken to spin up users (sec)"`
+	AverageTimeToCreateResources      FloatFormat `json:"Average Time taken to Create Resources (sec)"`
+	AverageTimeToRunPipelines         FloatFormat `json:"Average Time taken to Run Pipelines (sec)"`
+	UserCreationFailureCount          int64       `json:"Number of times user creation failed"`
+	UserCreationFailurePercentage     FloatFormat `json:"User creation failed percentage"`
+	ResourceCreationFailureCount      int64       `json:"Number of times resource creation failed"`
+	ResourceCreationFailurePercentage FloatFormat `json:"Resource creation failed percentage"`
+	PipelineRunFailureCount           int64       `json:"Number of times pipeline run failed"`
+	PipelineRunFailurePercentage      FloatFormat `json:"Pipeline run failed percentage"`
+}
```

Comment:
@naftalysh Since the intend of this file is to be read by machine, please use the naming of the fields in the JSON more machine friendly (do not use the human readable statementes meant for log output that can change easily) - let's keep the actual names independent of the log output and simple enough for the JSON to be processed easily (no whitespace, no brackets, etc.). Thanks

```suggestion
type LogData struct {
	Timestamp                         string      `json:"timestamp"`
	MachineName                       string      `json:"machineName"`
	BinaryDetails                     string      `json:"binaryDetails"`
	NumberOfThreads                   int         `json:"threads"`
	NumberOfUsersPerThread            int         `json:"usersPerThread"`
	BatchSize                         int         `json:"threadBatchSize"`
	NumberOfUsers                     int         `json:"totalUsers"`
	LoadTestCompletionStatus          string      `json:"status"`
	AverageTimeToSpinUpUsers          FloatFormat `json:"createUserTimeAvg"`
	AverageTimeToCreateResources      FloatFormat `json:"createResourcesTimeAvg"`
	AverageTimeToRunPipelines         FloatFormat `json:"runPipelineTimeAvg"`
	UserCreationFailureCount          int64       `json:"createUserFailures"`
	UserCreationFailurePercentage     FloatFormat `json:"createUserFailureRate"`
	ResourceCreationFailureCount      int64       `json:"createResourcesFailures"`
	ResourceCreationFailurePercentage FloatFormat `json:"createResourcesFailureRate"`
	PipelineRunFailureCount           int64       `json:"runPipelineFailures"`
	PipelineRunFailurePercentage      FloatFormat `json:"runPipelineFailureRate"`
}
```

Reply:
Agreed, updated